app/main.py

```diff
@@ -1 +1,7 @@
-# write your code here
+def copy_file(command: str) -> None:
+    command = command.split()
+    if command[1] == command[2]:
```

Comment:
Use unpacking instead of indexes. Validate command lenght and command value (should be exactly "cp")